trulens_eval/trulens_eval/utils/display.py

```diff
@@ -0,0 +1,20 @@
+import pandas as pd
+
+
+def get_feedback_result(tru_record, feedback_name):
```

Comment:
put the types in. Also you should probably put the return types in in a lot of the functions above.